python/src/vector_of_kll.cpp

```diff
@@ -276,31 +272,31 @@ py::array vector_of_kll_sketches<T,C,S>::get_num_retained() const {
 
 // Gets the minimum value of each sketch
 // TODO: allow subsets of sketches
-template<typename T, typename C, typename S>
-py::array vector_of_kll_sketches<T,C,S>::get_min_values() const {
+template<typename T, typename C>
+py::array vector_of_kll_sketches<T, C>::get_min_values() const {
```

Comment:
do we want to stick with values() here or switch to items()? we can probably make the sketches hold py::objects eventually.

Reply:
missed this. will fix

Reply:
we instantiate vector_of_kll_sketches for int and float. perhaps we should keep "values" there